arduino/sketch/sketch.go

```diff
@@ -302,5 +302,5 @@ func GenBuildPath(sketchPath *paths.Path) *paths.Path {
 	}
 	md5SumBytes := md5.Sum([]byte(path))
 	md5Sum := strings.ToUpper(hex.EncodeToString(md5SumBytes[:]))
-	return paths.TempDir().Join("arduino-sketch-" + md5Sum)
+	return paths.TempDir().Join("arduino/arduino-sketch-" + md5Sum)
```

Comment:
To allow cross platform compatibility in regards of the path separation, please use
```suggestion
	return paths.TempDir().Join("arduino", "sketch-" + md5Sum)
```
Also let's simplify the path a bit, by removing the double "arduino"